src/main/java/seedu/address/model/task/Task.java

```diff
@@ -146,6 +146,15 @@ public boolean isSameTask(Task otherTask) {
                 && description.equalsIgnoreCase(otherTask.getDescription());
     }
 
+    /**
+     * Returns true if this task is marked as done.
+     *
+     * @return boolean indicating whether this task is done.
+     */
+    public boolean getIsDone() {
```

Comment:
I think we can already get is done through the `getStatus()` method. Maybe you can use that or  standardize the other methods to use this so that we don't have two methods for the same thing?